index.html

```diff
@@ -1,12 +1,454 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <link
+        href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800&family=Nunito:ital,wght@0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+        rel="stylesheet">
+    <link rel="stylesheet" href="./main.css">
+    <link rel="stylesheet" href="./responsive.css">
+    <title>Kmarkter</title>
 </head>
+
 <body>
-    <h1>Hello Kmarketer</h1>
+    <header class="header">
+        <nav class="navbar-container">
+            <h2 class="logo">Kmarketer</h2>
+            <div class="navbar-container__items-wrapper">
+                <div class="navbar-container__items">
+                    <div class="items">
+                        <span class="items__item">Home</span>
+                        <span class="items__item">Products</span>
+                        <span class="items__item">Pricing</span>
+                        <span class="items__item">Contact</span>
+                    </div>
+                    <div class="navbar-container__icons">
+                        <box-icon name='search'></box-icon>
+                        <box-icon name='cart-alt'></box-icon>
+                    </div>
+                </div>
+                <box-icon name='menu-alt-right'></box-icon>
+            </div>
+            <div class="navbar-container__icons-sm">
+                <box-icon name='search'></box-icon>
+                <box-icon name='cart-alt'></box-icon>
+                <box-icon name='menu-alt-right'></box-icon>
+            </div>
+        </nav>
+        <div class="navbar-container__menu-items-sm">
```

Comment:
`navbar-container__menu-items-sm` is positioned outside the Block `(navbar-container)`.